skrf/io/touchstone.py

```diff
@@ -507,18 +507,18 @@ def get_noise_data(self):
 
     def is_from_hfss(self):
         """
-        Check if the Touchstone file has been produced by HFSS.
+        Check if the Touchstone might have complex port impedances for each frequency.
 
         Returns
         -------
         status : boolean
-            True if the Touchstone file has been produced by HFSS
+            True if the Touchstone file might have complex port impedances for each frequency.
             False otherwise
         """
         if self.comments is None:
             return False
 
-        if 'exported from hfss' in str.lower(self.comments):
+        if 'hfss' in str.lower(self.comments) or 'not renormalized' in str.lower(self.comments):
```

Comment:
note that you could catch a comment mentioning hfss, but not necessarily a file having (that was true before of course, but a bit less probable)

Reply:
It shouldn't matter since the function reading the port impedances will return the reference impedance if it doesn't find them. It would be better for this function to actually check for the impedances instead, since it's used to check if the file might have port impedances.